### PR TITLE
add strange glass and pane

### PR DESCRIPTION
**Pull request by pentantan**

### CODE REVIEW

**Review by Jonathing — APPROVED** [review]

**Comment by Jonathing** [comment]
Ignore the workflow checks for the pull request. It should build normally next time.

File: src/main/java/midnight/data/models/ItemModelTable.java [hunk 1]
```diff
@@ -190,6 +190,9 @@ public static void registerItemModels(BiConsumer<Item, IModelGen> c) {
 
         register(MnItems.DARK_STICK, item -> generated(name(item, "item/%s")));
 
+        register(MnBlocks.STRANGE_GLASS, item -> inherit(name(item, "block/%s")));
+        register(MnBlocks.STRANGE_GLASS_PANE, item -> generated(name(item, "block/strange_glass")));
```
Comment (ShadewEnder) [inline src/main/java/midnight/data/models/ItemModelTable.java:194]: You might wanna try `name(item, "block/%s", "_pane")` next time, it removes the given suffix from the dynamic block name so we don't need to update when the name changes

File: src/main/java/midnight/common/item/MnItems.java [hunk 2]
```diff
@@ -191,6 +191,10 @@
     public static final Item DARK_PEARL_ORE = item(MnBlocks.DARK_PEARL_ORE, MnItemCategory.ORES, MnItemGroup.BLOCKS);
     public static final Item DARK_PEARL_BLOCK = item(MnBlocks.DARK_PEARL_BLOCK, MnItemCategory.MINERAL_BLOCKS, MnItemGroup.BLOCKS);
 
+    //Strange Glass
```
Comment (ShadewEnder) [inline src/main/java/midnight/common/item/MnItems.java:194]: Little formatting error: there's no space at the start of the comment

Comment (ShadewEnder) [inline src/main/java/midnight/common/item/MnItems.java:194]: Little formatting request: `//Strange Glass` -> `// Strange Glass`

File: src/main/java/midnight/common/block/MnBlocks.java [hunk 3]
```diff
@@ -330,6 +334,9 @@ public static void setupRenderers() {
         RenderTypeLookup.setRenderLayer(BLOOMCRYSTAL, RenderType.getCutout());
         RenderTypeLookup.setRenderLayer(CRYSTAL_FLOWER, RenderType.getCutout());
 
+        RenderTypeLookup.setRenderLayer(STRANGE_GLASS, RenderType.getTranslucent());
+        RenderTypeLookup.setRenderLayer(STRANGE_GLASS_PANE, RenderType.getTranslucent());
```
Comment (ShadewEnder) [inline src/main/java/midnight/common/block/MnBlocks.java:338]: These must be cutout, the texture isn't translucent

**Review by ShadewEnder — CHANGES_REQUESTED** [review]
Looks good! It's almost complete: there's two very little changes that I'd still like to see, I commented them on the commits. If you fix these your PR is complete and we can pull it.

File: src/main/java/midnight/common/item/MnItems.java (hunk 2 above)
Comment (ShadewEnder) [reply src/main/java/midnight/common/item/MnItems.java:194]: I know this is a stupid request but I strive for perfection :)

**Review by ShadewEnder — APPROVED** [review]
Perfect! Thanks for the work!

**Comment by ShadewEnder** [comment]
I won't wait for the actions to complete, for some reason they don't work right now

**Comment by pentantan** [comment]
> Perfect! Thanks for the work!

👍

**Pull request merged**